ui/nuxeo-data-table/data-table-checkbox.js

```diff
@@ -48,8 +50,31 @@ import '../widgets/nuxeo-checkmark.js';
           reflectToAttribute: true,
           value: false,
         },
+        ariaLabel: {
+          type: String,
+          reflectToAttribute: true,
+          computed: '_computeLabel(i18n)',
+          observer: '_ariaLabelChanged',
+        },
       };
     }
+
+    ready() {
+      super.ready();
+      this.setAttribute('tabindex', 0);
+    }
+
+    _ariaLabelChanged(newValue, oldValue) {
+      const ariaLabel = this.$.checkmark.getAttribute('aria-label');
+      // Don't stomp over a user-set aria-label.
+      if (!ariaLabel || oldValue === ariaLabel) {
```

Comment:
Shouldn't we be checking for the aria label on `this` instead of `this.$.checkmark`? Because this.$.checkmark will always have a `aria-label` set, right?

Also, do we need to set it both on the `nuxeo-checkmark` element and the host? isn't it just enough to set on the host?

Reply:
We ended up removing this method since we found out this strategy wouldn't really allow users to override the `aria-label`. Thus, in this PR, we are setting the `aria-label` directly in `nuxeo-checkmark` with a predefined label (in this case `command.select`).